File: lib/models/programs_info.dart
import 'package:udevs_video_player/models/tv_program.dart';

class ProgramsInfo {

  ProgramsInfo({
    required this.day,
    required this.tvPrograms,
  });
  String day;
  List<TvProgram> tvPrograms;

  Map<String, dynamic> toJson() {
    final map = <String, dynamic>{};
    map['day'] = day;
    map['tvPrograms'] = tvPrograms.map((v) => v.toJson()).toList();
    return map;
  }

  @override
  String toString() => 'ProgramsInfo{day: $day, tvPrograms: $tvPrograms}';
}

File: android/src/main/kotlin/uz/udevs/udevs_video_player/models/configuration/ProgramsInfo.kt
package uz.udevs.udevs_video_player.models.configuration

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class ProgramsInfo(
    @SerializedName("day")
    val day: String,
    @SerializedName("tvPrograms")
    val tvPrograms: List<TvProgram>,
) : Serializable